setup.md

```diff
@@ -27,9 +27,6 @@ In the lesson, you will find out how to access the data in this folder.
 > or [(KDE) Konsole](https://konsole.kde.org/)
 > or [xterm](https://en.wikipedia.org/wiki/Xterm),
 > which can be found via the applications menu or the search bar.
-> 
-> To check if your machine is set up to use something other than Bash, type `echo $SHELL` in your terminal window.
->
 > If your machine is set up to use something other than Bash, you can run it by opening a terminal and typing `bash`.
```

Comment:
This too please :)

Reply:
I'm seeing two different contexts for this comment. One shows the sentences for checking your shell on line 31. Another shows that line removed, and the "If your machine is set up to use something other than Bash" line is line 30, a sentence that dates back 2 years. [My current fix-1033 branch](https://github.com/mikerenfro/shell-novice/blame/fix-1033/setup.md#L30) shows the latter.

Are you suggesting removing the "If your machine is setup to use something other than Bash" line, or were you referring to the line showing how to check if the shell was set to Bash?